### PR TITLE
remove support for process ngsi V1 notifications

### PR DESCRIPTION
**Pull request by AlvaroVega**
related with issue https://github.com/telefonicaid/perseo-fe/issues/714
- [x] Update to v2 notifications from /test/data/good_notices/

Do not merge until CI tests would be updated
Do not merge until ~https://github.com/telefonicasc/iotp-pqa/pull/67~ https://github.com/telefonicasc/iotp-pqa/pull/78

### CODE REVIEW

File: test/unit/notices_Do.js [hunk 1]
```diff
@@ -84,53 +61,13 @@ var coreNotice1 = {
 
 describe('Notices Do', function() {
     describe('#DoNotice', function() {
-        var v1notice, v2notice;
+        var /*v1notice,*/ v2notice;
 
         beforeEach(function() {
-            v1notice = JSON.parse(noticeExampleV1);
+            //v1notice = JSON.parse(noticeExampleV1);
```
Comment (fgalan) [inline test/unit/notices_Do.js:67]: Leftover to be deleted?

Comment (AlvaroVega) [reply test/unit/notices_Do.js]: done in 93cadb1a00a19a60be02cb02d173cac775f6a3e7

**Review by fgalan — APPROVED** [review]
LGTM

Pending of CI e2e fixes to be merged

**Comment by fgalan** [comment]
> Do not merge until ~[telefonicasc/iotp-pqa#67](https://github.com/telefonicasc/iotp-pqa/pull/67)~ [telefonicasc/iotp-pqa#78](https://github.com/telefonicasc/iotp-pqa/pull/78)

That PR has been merged.

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**